### ISSUE TITLE
The Cake Is A Lie!

### ISSUE THREAD

**Issue by ironapple09** [titaniumnetwork-dev/Donglalu-Proxy-The-Second-Coming#3]
The year is 4712, and the people of the planet Procul Hinc are exploring the planet that previously had been known as Earth. Earth’s natives went extinct in the year 2146 due to climate changes inhibiting plant growth. What you’re to hear is what occurred on the day in question.

“Hey Vos, I think I’ve found something”
“Really? What is it, Sunt?”

Then there is a moment of silence. The search party had broken up into pairs earlier in the day. Vos and Sunt were some of the first to find anything, and as such, they were doubtful of it’s value.

Sunt dares to interrupt the silence. “It seems to be… some sort of an electronic device? I think this is an electrical cable, I remember reading about these in grade school!”
Vos, still comprehending the situation, makes a suggestion or two. “Wow, we should take this to our vehicle and plug it into our generator — I knew this would come in useful!”

They then proceed to take the device to their vehicle. The Procullians are not known for their strength, and in fact thrive in an environment where automation does all of the heavy lifting. However, here on Earth, such technology is not in the area.

Upon plugging in the device, nothing immediately happens. Slightly disappointed, Vos assumes that it must no longer be in working condition. “Why would it work, anyways,” he thinks to himeslf, “it’s probably three thousand years old.”

“Vos! Look here!”

Sunt has discovered what appears to be a button.

“How old-fashioned! I wonder if pressing it will cause the system to—“

Sunt pressed the button, only to be interrupted by a high pitched, loud, double-beep. Then some fans start spinning up, and the shattered old display lights up.

“Holy shit, it’s powering up” the both of them exclaim.

Minutes later, they find that the system has booted up; apparently, the human who once operating the system set it up to automatically login upon boot up. After some poking around, Vos and Sunt find how to view the system information.

The system information, clearly inaccurately stating that the system is an Apple product (despite being labelled “Dell” on the side panel), clarifies that, indeed, this system features the eternal CPU that Sunt remembers hearing so much about.

It featured the legendary Intel Core 2 Duo.

Dual core, do more!

**Comment by truubo** [comment]
TLdr please

**Comment by ironapple09** [comment]
No

**Comment by Enstrayed** [comment]
Why the hell do you have the entire transcript of what seems like an Intel ad saved somewhere that you can conveniently post it in a GitHub issue??

**Comment by ironapple09** [comment]
@Enstrayed its not even a intel ad, its some random thing frank128 made for a hackintosh post June 2021

**Comment by Enstrayed** [comment]
@ironapple09 Why am I not surprised
